packages/repack/commands.js

```diff
@@ -50,6 +50,117 @@ function getCommands() {
   return commands;
 }
 
+const startCommandOptions = [
+  {
+    name: '--cert <path>',
+    description: 'Path to custom SSL cert',
+  },
+  {
+    name: '--host <string>',
+    description: '(default: "")',
+  },
+  {
+    name: '--http',
+    description: 'Enables https connections to the server',
+  },
+  {
+    name: '--key <path>',
+    description: 'Path to custom SSL key',
+  },
+  {
+    name: '--port <number>',
+    description: 'The port number that runs the server on',
+  },
+  {
+    name: '--no-interactive',
+    description: 'Disables interactive mode',
+  },
+  {
+    name: '--silent',
+    description: 'Silents all logs to the console/stdout',
+  },
+  {
+    name: '--experimental-debugger',
+    description:
+      "[Experimental] Enable the new debugger experience and 'j' to debug. This enables the new frontend experience only: connection reliability and some basic features are unstable in this release.",
+  },
+  {
+    name: '--verbose',
+    description: 'Enables verbose logging',
+  },
```

Comment:
```suggestion

```

`--verbose` is automatically added by RNC CLI, right now `--verobse is presented two times in the `--help` output.